tests/unit/test_server.py

```diff
@@ -53,23 +53,23 @@ def test_install_server(self, mock_execute, mock_check_rpm):
         mock_execute.assert_called_with(server.deploy_install_configure,
                                         local_path, hosts=get_host_list())
 
+    @patch('prestoadmin.server.sudo')
     @patch('prestoadmin.server.package.deploy_install')
     @patch('prestoadmin.server.update_configs')
-    def test_deploy_install_configure(self, mock_update, mock_install):
+    def test_deploy_install_configure(self, mock_update, mock_install,
+                                      mock_sudo):
         local_path = "/any/path/rpm"
+        mock_sudo.side_effect = self.mock_failed_then_succeeded()
         server.deploy_install_configure(local_path)
         mock_install.assert_called_with(local_path)
         mock_update.assert_called_with()
```

Comment:
If we're not checking the arguments used to call the mock maybe we can substitute this call with something simpler?